CodeEdit/Documents/CodeEditDocumentController.swift

```diff
@@ -41,7 +41,7 @@ class CodeEditDocumentController: NSDocumentController {
 }
 
 extension NSDocumentController {
-    func openDocument(completionHandler: @escaping (NSDocument?, Bool) -> Void) {
+    func openDocument(completionHandler: @escaping (NSDocument?, Bool) -> Void, cancelHandler: @escaping () -> Void) {
```

Comment:
Instead of

```swift
func openDocument(completionHandler: @escaping (NSDocument?, Bool) -> Void, cancelHandler: @escaping () -> Void) {}
```

consider using a blank or `onCompletion` label for the first argument and `onCancel` for the second one.

```swift
func openDocument(_ completionHandler: @escaping (NSDocument?, Bool) -> Void, onCancel: @escaping () -> Void) {}
```